README.md

```diff
@@ -17,11 +17,11 @@ Inputs
 ------
 
 - `vswhere`: Path to `vswhere.exe` (default system-installed copy).
-- `arch`: Build architecture (default `amd64`).
-- `host_arch`: Host architecture override.
+- `host_arch`: Host architecture override (defaults to the processor architecture).
```

Comment:
This is the build architecture not the host architecture right? This is where the compiler is running.

Reply:
This is following vsdevenv terminology. The host architecture is where the compiler is running and the (target) architecture is where the compiled binaries will run. We shouldn't bring llvm terminology in here

Reply:
That is the de facto standard terminology, I don't think that we should be using something else.

Reply:
It is not standard on Windows; Visual Studio uses `arch` and `host_arch`. This action is about calling vsdevcmd which uses those terms, and the arguments to this action map one-to-one to those command line parameters. Using anything else would be confusing.

```
C:\Code>"c:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\vsdevcmd.bat" /help
.
** Visual Studio "17" Developer Command Prompt Help **
** Version :
.
Syntax: vsdevcmd.bat [options]
[options] :
    -arch=architecture : Architecture for compiled binaries/libraries
           ** x86 [default]
           ** amd64
           ** arm
           ** arm64
    -host_arch=architecture : Architecture of compiler binaries
           ** x86 [default]
           ** amd64

Reply:
Ping @compnerd !

Reply:
I understand that it may not be the terminology that VS uses, but this is the standard terminology almost universally for cross-compilation. I think that we should be pretty explicit that the use here is incorrect because otherwise it is easy for users to get confused. I think that leaving the parameters as is, but documenting it large bold print that this is the incorrect terminology usage is important.

Reply:
Updated to more neutral "host_arch: determines the value of the host_arch argument to vcvarsall".